config/settings.yml

```diff
@@ -264,8 +264,6 @@ modules_appeals_api:
       # List of flags to indicate parts of documentation that are still WIP and not ready for production.
       # See the rswag:appeals_api:dev rake task, spec_helper, and DocHelpers#wip_doc_enabled? for usage info to opt-in.
       # NOTE: Do not remove this key even if it's empty.
```

Comment:
Judging by the note and future considerations, I'm assuming we want the `wip_docs` key to stay around?

Reply:
Correct. It should stick around for the inevitable time we have some other docs in WIP state that we want in the lower envs but not production.